src/welcome-channel.yaml

```diff
@@ -3,29 +3,24 @@
     Follow our Code of Conduct: The [Code of Conduct](https://github.com/PrismLauncher/PrismLauncher/blob/develop/CODE_OF_CONDUCT.md) applies not only on GitHub but in all Prism Launcher community spaces. Those who do not intend to follow the Code of Conduct should not interact in Prism spaces.
     Use English: The common language in our community is English, but we understand that not everyone speaks English at the same level. Try to speak English to the best of your ability, and don't worry if you need to use translation tools. We will not tolerate making fun of people for poor English.
     Listen to and respect all community team members: Please take requests from community team members seriously and listen when one asks you to do something. If you have a problem with what a team member is asking you to do, we advise that you step away from the situation and send a private message to another team member.
-    Keep all activity legal, legitimate, and appropriate: We do not tolerate users breaking any laws, breaching a Terms of Service or an EULA, sexual content, or anything the moderators determine to be malicious or inappropriate.
-    Don't spam and don't scam: No advertisement outside of the showcase channels. Also, no NFTs.
-    Follow the given channel topic: Don't derail a discussion, and don't start conversations outside of the channel in which they're most appropriate. For example, if you start to get off-topic in an on-topic channel, ask other participants to move to an off-topic channel.
-    When requesting help, specify all relevant information and provide error logs: Help us to help you! Without this information, we won't be able to help with your issue as easily or as quickly.
+    Do not harass anyone: We want to make participation in our community a harassment-free experience for everyone. If you think that someone is breaking the rules, contact the moderators, instead of harassing them.
+    Keep all activity legal, legitimate, and appropriate: We do not tolerate users breaking any laws, breaching a ToS or an EULA, posting sexual content, or anything the moderators determine to be malicious or inappropriate. Shilling crypto currency related products and gambling websites is strictly forbidden.
+    Follow the given channel topic: Don't derail a discussion, and don't start conversations outside of the channel in which they're most appropriate. For example, if you start to get off-topic in an on-topic channel, ask other participants to move to an off-topic channels. Advertisements are not allowed outside of showcase channels.
     Don't import drama or controversy from other communities: In these topics, emotions run high and opinions run even higher. To avoid arguments and other similar problems, simply avoid discussion of controversial topics from other communities.
-    Leave other launcher developers alone: We will not tolerate bashing on the developers of other launchers, including those which we have forked from.
```

Comment:
That one should be kept

Reply:
I thought this is already implied with the "Do not harass anyone" rule. Maybe I can add another sentence about this there.